### ISSUE TITLE
Confusing ambiguity of page titles, missing section names

### ISSUE THREAD

**Issue by JohnVeness** [godotengine/godot-docs#6971]
**Your Godot version:**
4.0.

**Issue description:**
At the moment, the title at the top of the page content, and the HTML page title, show only the page name, not which section it is in. This means, for example, that there are two pages called Introduction. If you didn't have the left-hand navigation open, you wouldn't know at a glance which one you were reading.

**URL to the documentation page (if already existing):**
Compare https://docs.godotengine.org/en/stable/about/introduction.html : 

![image](https://user-images.githubusercontent.com/2512915/224571407-946d2093-2896-4983-a290-fd736cbb9985.png)

and https://docs.godotengine.org/en/stable/getting_started/introduction/index.html :

![image](https://user-images.githubusercontent.com/2512915/224571259-5c831391-5553-4f34-95d4-5ed492a0feca.png)

Could the section title be included at the top (and similar in the HTML title), e.g. [Home icon] / About / Introduction, and [Home icon] / Getting Started / Introduction?

**Comment by YuriSizov** [comment]
> Could the section title be included at the top

Yeah, should be possible, I think. We customize that template already.

**Comment by tetrapod00** [comment]
Besides the general issue with displaying the breadcrumbs, for this particular case, we should also think about streamlining the experience of reading the Godot docs from the beginning.

Depending on how you count, there are maybe four Introduction pages that you will encounter, if you start from the beginning and read in order (even if you skip things like most of the About section):
https://docs.godotengine.org/en/stable/index.html
https://docs.godotengine.org/en/stable/about/introduction.html
https://docs.godotengine.org/en/stable/getting_started/introduction/index.html
https://docs.godotengine.org/en/stable/getting_started/introduction/introduction_to_godot.html
